app/models/concerns/sample_sheet.rb

```diff
@@ -28,11 +28,33 @@ def barcode_set
     libraries.first.tag.tag_set.uuid
   end
 
+  # Sample bio Name field - TEMPORARY: Want well row to have sample_names when isoSeq tags
+  def find_sample_name
+    if defined?(sample_names) # When row type is well
```

Comment:
Won't this also hide sample names with untagged wells?
I'm also a little concerned that if someone adds 'sample_names' to library, this will change the behaviour here (and as a result we'll also be dropping sample names for untagged libraries).

Reply:
This fix is a temporary fix until we are able to determine the pipeline (type) of a sample e.g isoseq. Also sample_names should never be added to library as librarys can only have one sample.
Im not sure I understand, the intended functionality is: 

On the csv well row: If any libraries in the well are not tagged or are isoseq tagged then the method check_library_tags will return false and therefore sample_names would be returned.
On the library (sample) rows: the request sample_name is returned

Reply:
Yep... that looks like it matches up.